### DISCUSSION TITLE
hono/client is undocumented?

### DISCUSSION THREAD

**Discussion by Meess** [honojs/hono#3060] [Q&A] [answered]
I feel that hono/client is super powerful and enables a real full stack experience through supporting the frontend with a fully typed version of the backend that is easy to use. And not many other frameworks have it (or did I also mis it there?)

However I have noticed it quite late, and also now trying to search for it in the documentation I can't find it mentioned in de documentation. Is it such a small feature that it's less relevant? Coming from Springboot / Django / Ruby on Rails with mainly REST API's, did I miss something? Is there new tooling for it (I get that there is GraphQL, which has it's own merits, or tRPC although not working well with non typescript API's), but still think this is a foundational feature that not much other frameworks offer in the Node eco-system have. Curious what others think of it

**Comment by yusukebe** [comment, marked as answer]
Hi @Meess 

Have you been able to check this page?

https://hono.dev/docs/guides/rpc#client

It does not have an individual page for `hono/client`, but it is written there. We might have to create the page `hono/client` and move some descriptions to it.

**Reply by Meess** [reply]
Not sure how I missed that...
I would be in favour of hono/client as a specific page, fully focussed on the client to really draw the focus on the supported client side, though I might also have to work on my reading skills 👀

**Comment by yusukebe** [comment]
Hey @NicoPlyley !

What do you think about creating a `hono/client` specific page?

**Reply by NicoPlyley** [reply]
How would you propose this be done? Splitting RPC to show how to set it up on the server and client will show just frontend? Or would you be looking for more an API section going through all the options? 

Either way I don't think there are any issues having more information as long as it stays organized.

**Reply by yusukebe** [reply]
> Either way I don't think there are any issues having more information as long as it stays organized.

You are right. It's fine since we have the client description on [the RPC page](https://hono.dev/docs/guides/rpc). Let's leave it as it is now. Thank you.
